src/github.ts

```diff
@@ -18,7 +18,7 @@ const GitHub = (githubToken: string, fetchFn = fetch) => {
     const requestParams = {
       method: method,
       headers: {
-        Accept: "application/vnd.github.squirrel-girl-preview+json",
+        Accept: "application/vnd.github.squirrel-girl-preview+json, application/vnd.github.v3.html+json",
```

Comment:
Took me so [many hours to figure this out before for danger-js](https://github.com/danger/danger-js/blob/master/source/platforms/github/_tests/_github_api.test.ts#L116-L137)

Reply:
Ouch, good catch!